clusters/nerc-ocp-infra/logging/application.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: logging
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/ocp-on-nerc/nerc-ocp-config.git
+    targetRevision: HEAD
+    path: logging/overlays/nerc-ocp-infra
+  destination:
+    name: nerc-ocp-infra
+    namespace: openshift-logging
```

Comment:
This namespace is in PR https://github.com/OCP-on-NERC/nerc-ocp-config/pull/70 

But that namespace will be created by this argocd App, but we can't create this argocd app because it will look for the `openshift-logging` namespace to begin with. 

I am not sure sure about how much the namespace matters; we could put all these apps into the `openshift-gitops` namespace just so we have all the argocd apps in one namespace. 

Or we should be creating the namespace (and other clusterwide things) using the cluster-scope app which already exists. That's what we should be doing regardless of this situation. 

@larsks thoughts?

Reply:
These should live in their own namespace, which should be created via a PR to the `nerc-ocp-config` repository before this one merges.

Reply:
These should live in their own namespace, which should be created via a PR to the `nerc-ocp-config` repository before this one merges.